src/write.rs

```diff
@@ -583,21 +618,24 @@ impl<S: Span> Report<'_, S> {
                                     .as_ref()
                                     .map_or(true, |m| !std::ptr::eq(ll.label, m.label))
                         })
-                        .find(|(j, ll)| {
-                            ll.col == col && ((row <= *j && !ll.multi) || (row <= *j && ll.multi))
-                        })
```

Comment:
Another instance of logic that can be simplified. This was also proposed by Clippy.